### PR TITLE
Add support for DIN rail relay _TZ3000_qeuvnohg

### PR DESCRIPTION
**Pull request by ManarINW**
Added support for the DIN rail relay model AT-Q-SY1-JZT with the manufacturer name _TZ3000_qeuvnohg.

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -3565,16 +3566,23 @@
...
+              57345
             ],
             "bindings": [
               6,
               1794,
-              2820
+              2820,
+              1026,
+              57344,
+              57345
```
Comment (Boechie) [inline app.json]: What do those added IDs change in functionality? What do they mean?

File: drivers/smartplug/driver.compose.json [hunk 2]
```diff
@@ -68,8 +69,8 @@
     ],
     "endpoints": {
       "1": {
-        "clusters": [0, 4, 5, 6, 1794, 2820],
-        "bindings": [6, 1794, 2820]
+        "clusters": [ 0, 3, 4, 5, 6, 1794, 2820, 1026, 57344, 57345 ],
+        "bindings": [ 6, 1794, 2820, 1026, 57344, 57345 ]
```
Comment (Boechie) [inline drivers/smartplug/driver.compose.json]: What do those added IDs change in functionality? What do they mean?

**Review by Boechie — COMMENTED** [review]
What do those added IDs change in functionality? What do they mean?

**Comment by JohanBendz** [comment]
Hi guys, please see "mail" thread in the Athom community.

**Pull request closed without merging**